front-end/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "36-full-stack-crud",
+  "version": "1.0.0",
+  "description": "![cf](http://i.imgur.com/7v5ASc8.png) 36: Async Actions ===",
+  "main": "webpack.common.js",
+  "scripts": {
+    "watch": "webpack-dev-server --config webpack.dev.js",
+    "test": "jest"
```

Comment:
add lint script and fix your linter errors but running `npm run lint` in your terminal

```json
"lint": "eslint ."
```